charts/node/values.yaml

```diff
@@ -248,12 +248,29 @@ node:
     # -- List of Relay Chain RPCs to connect
     relayChainRpcUrls: []
 
+  # -- EXPERIMENTAL!!!
+  # Run the collator node without a relay chain via light client
+  # ref: https://github.com/paritytech/cumulus/pull/2270
+  # Enabling this option will disable the values of collatorRelayChain
+  lightClientMode:
```

Comment:
Can you enable both the `externalRelayChain` and `lightClientMode` features together?

If they are mutually exclusive, I recommend linking the `externalRelayChain` and `lightClientMode` features together, as they serve the same purpose:

```yaml
{{ if externalRelayChain.enabled }} 
  {{ if externalRelayChain.relayChainRpcUrls }}
    --relay-chain-rpc-urls {{- range externalRelayChain.relayChainRpcUrls }} "{{ . }}" {{- end }} \                
  {{ else }}
    --relay-chain-light-client \ 
  {{ end }}
{{ end }}
```

Reply:
I added a different logic here. The `externalRelaychain`, `lightClient` and even the regular collator with relaychain are just different modes to run a collator.